downstream/modules/hub/proc-adding-an-execution-environment.adoc

```diff
@@ -1,26 +1,32 @@
 
 [id="adding-an-execution-environment"]
 
-= Adding an {ExecEnvShort}
+= Adding and signing an {ExecEnvShort}
 {ExecEnvNameStart} are container images that make it possible to incorporate system-level dependencies and collection-based content. Each {ExecEnvShort} allows you to have a customized image to run jobs, and each of them contain only what you need when running the job.
 
 .Procedure
 . From the navigation panel, select {MenuACExecEnvironments}.
 
-. Click btn:[Create execution environment].
+. Click btn:[Create execution environment] and enter the relevant information in the fields that appear.
 
-. Enter the name of the {ExecEnvShort}.
+.. The *Name* field displays the name of the {ExecEnvShort} on your local registry.
 
-. Enter the upstream name.
+.. The *Upstream name* field is the name of the image on the remote server.
 
-. Under *Registry*, select the name of the registry from the drop-down menu.
+.. Under *Registry*, select the name of the registry from the drop-down menu.
 
-. Enter tags in the *Add tag(s) to include* field.
+.. Optional: Enter tags in the *Add tag(s) to include* field.
 If the field is blank, all the tags are passed.
 You must specify which repository-specific tags to pass.
 
-. Optional: Enter tags to exclude in *Add tag(s) to exclude*. 
+.. Optional: Enter tags to exclude in *Add tag(s) to exclude*. 
```

Comment:
Consider adding 'the' before the UI element, and 'field' after the UI element, as mentioned in other points.